vanilla-framework/vanilla-framework
# Update docs/media object page content

## Done
- Updated copy 👍 
- Updated demo example to showcase live example 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1BaY3k01-6zvhC67PMxZD44IKHs4_pTpPm_ix98rHW98/edit#heading=h.alqwiio0lnlr)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/patterns/media-object
- Pull code
- Run ./run
- Review updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object-circ-img/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object-large/
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object/

## Details
- Fixes https://github.com/ubuntudesign/vanilla-design/issues/358

## Screenshot
![Screenshot 2019-03-27 at 17 19 35](https://user-images.githubusercontent.com/17748020/55097814-865ba880-50b4-11e9-95eb-c682d5e24d35.png)
